README.md

```diff
@@ -2,4 +2,52 @@
 
 *⚠️ Note: This repository is a WIP, "watch" it to keep up with updates!*
 
-This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/).
+## Project description
+
+This repository hosts Bokeh equivalents for various plots from ["Fundamentals of Data Visualization" by Claus O. Wilke](https://clauswilke.com/dataviz/). It provides a collection of interactive data visualizations implemented using the Bokeh library.
+
+## Table of Contents (WIP)
```

Comment:
```suggestion
## Table of contents (WIP)
```

Let's use sentence case for headings per our [docs style guide](https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#principles-of-good-style).

Reply:
I'm also wondering if we can leave this section blank and add things here as we add notebooks to the repo?

Reply:
OK. That makes sense.